Content.Client/Pinpointer/UI/NavMapControl.cs

```diff
@@ -236,6 +240,9 @@ protected override void KeyBindUp(GUIBoundKeyEventArgs args)
             // Clear current selection with right click
             if (TrackedEntitySelectedAction != null)
                 TrackedEntitySelectedAction.Invoke(null);
+
+            // Toggle beacon labels
+            _beacons.Pressed = !_beacons.Pressed;
         }
```

Comment:
It's not a good idea to have two different functions tied to the same player input. I'd remove this

Reply:
I'll see if I can find a better keybind for the toggle but if I dont, then I would rather remove the unselect function. When does anyone ever need/try to untarget something on the map view? Does anyone have any recollection of ever using that function?

Reply:
I moved it to Examine